src/site/xdoc/userguide.xml

```diff
@@ -60,6 +60,21 @@
 
       <subsection name ="Usage">
         <ol style="list-style-type: decimal">
+          <h4>Prerequisites</h4>
+          <p>
+            Commons Crypto relies on standard JDK 7 (or above) and OpenSSL 1.0.1c (or above) for production
```

Comment:
Hi @winningsix, thanks for you review.  JDK version was changed by haifeng  from 6 to 7 in this [commit](https://github.com/apache/commons-crypto/commit/32f789dbff523117223532ac8adfd5cacd1d350a)